source/_integrations/weatherflow.markdown

```diff
@@ -60,4 +60,22 @@ Additionally the following diagnostic sensors are available:
 
 ## Networking notes
 
-This {% term integration %} relies on the ability of Home Assistant to receive `UDP` traffic on port `50222`. You may run into trouble if you have a more complex network setup utilizing either VLANs or multiple subnets.
+This {% term integration %} relies on the ability of Home Assistant to receive `UDP` traffic on port `50222`. You may run into trouble if you have a more complex network setup utilizing either VLANs or multiple subnets. If you are deploying Home Assistant via a Docker container, make sure to run using host networking mode or expose port `50222` as `UDP`. For example.:
+
+```
+docker run -d \
+   ...
+  --network=host \
+  ghcr.io/home-assistant/home-assistant:stable
+```
+
+or
+
+```
+docker run -d \
+   ...
+  -p 50222:50222/udp \
```

Comment:
This should make no difference, as Home Assistant must run on the host network.

Could you elaborate on this change?

../Frenck

Reply:
Aah I see now, this is trying to add instruction for the non-hostwork case.

Unfortunately, the Home Assistant project does not support running without hostnetwork support. Therefore, this should be in our official documentation.

../Frenck

Reply:
Understood it's preferred to keep the docs focused on the supported configuration. Would you be opposed to an addition that states:

"If you are deploying Home Assistant via a Docker container, make sure to run using host networking mode. You may see the error `Config flow could not be loaded: Unknown error` if Home Assistant is unable to receive `UDP` traffic on port `50222`.

I could be helpful to add some SEO discoverability for the specific issue given the error message is a generic one. Not sure if its practical to actually catch this error in the actual integrations code, preempting the generic error.

Reply:
We do not support that use case. The correct solution is to install Home Assistant as documented.